Formula/c/cattle.rb

```diff
@@ -1,7 +1,7 @@
 class Cattle < Formula
   desc "Brainfuck language toolkit"
   homepage "https://kiyuko.org/software/cattle"
-  license "GPL-2.0"
+  license "GPL-2.0-or-later"
```

Comment:
https://git.kiyuko.org/cattle/tree/COPYING

Reply:
yeah, I did in here https://github.com/Homebrew/homebrew-core/pull/177245#issuecomment-2226948624